src/signer_node/message_processor/mod.rs

```diff
@@ -40,6 +56,141 @@ fn get_valid_block(state: &NodeState, blockhash: Hash) -> Result<&Block, Error>
     }
 }
 
+/// Create own VSSs and send to each other signers.
+/// Returns
+///     * own random key pair
+///     * a VSS for itself(for positive and negative)
+///     * own commitments
+pub fn create_block_vss<T, C>(
```

Comment:
this function should be move process_blockvss.rs ?

Reply:
No, it is used in message_processor/mod.rs and process_candidateblock.rs.

On my ideal, it should be in out of message_processor module because cryptography code should be independent of presentation module such as message_processor.